doc/users/text_props.rst

```diff
@@ -59,3 +60,93 @@ upper right.
+|                     | 'fantasy', 'monospace', 'sans', 'sans serif',      |
+|                     | 'sans-serif', 'serif'}``.                          |
+|                     |                                                    |
++---------------------+----------------------------------------------------+
+|  ``'font.style'``   | The default style, ex ``'normal'``,                |
+|                     | ``'italic'``.                                      |
+|                     |                                                    |
++---------------------+----------------------------------------------------+
+| ``'font.variant'``  | Default variant, ex ``'normal'``, ``'small-caps'`` |
+|                     | (untested)                                         |
++---------------------+----------------------------------------------------+
+| ``'font.stretch'``  | Default stretch, ex ``'normal'``, ``'condensed'``  |
+|                     | (incomplete)                                       |
+|                     |                                                    |
++---------------------+----------------------------------------------------+
+|  ``'font.weight'``  | Default weight.  Either string or integer          |
+|                     |                                                    |
+|                     |                                                    |
++---------------------+----------------------------------------------------+
+|   ``'font.size'``   | Default font size in points.  Relative font sizes  |
+|                     | (``'large'``, ``'x-small'``) are computed against  |
+|                     | this size.                                         |
++---------------------+----------------------------------------------------+
+
+The mapping between the family aliases (``{'cursive', 'fantasy',
+'monospace', 'sans', 'sans serif', 'sans-serif', 'serif'}``) and actual font names
+is controlled by the following rcParms
+
+
++----------------------------+----------------------------------------------------+
+| ``'serif'``                | ``'font.serif'``                                   |
++----------------------------+----------------------------------------------------+
+| ``'monospace'``            | ``'font.monospace'``                               |
++----------------------------+----------------------------------------------------+
+| ``'fantasy'``              |                   ``'font.fantasy'``               |
++----------------------------+----------------------------------------------------+
+|``'cursive'``               | ``'font.cursive'``                                 |
++----------------------------+----------------------------------------------------+
+| ``{'sans', 'sans serif',   | ``'font.sans-serif'``                              |
+| 'sans-serif'}``            |                                                    |
```

Comment:
Add the double backticks at end and start where the line was split in the left-hand box.